html_css/grid-lessons/advanced_grid_properties.md

```diff
@@ -56,7 +56,7 @@ Enter, `repeat()`.
 
 ### Repeat
 
-`repeat()` is a CSS function available to the CSS Grid template properties that allows us to define a number of rows or columns and the size we want them to be without having to manually type out each individual column's size. For example, in our setup above:
+`repeat()` is a CSS function available to the CSS Grid template properties that allows us to define a number of rows or columns and the size we want them to be without having to manually type out each individual column's or row's size. For example, in our setup above:
```

Comment:
Good catch here with the inclusion of row in addition to column for this example. I think it'd be best to actually use the term "track" here since that covers either a grid column or grid row. 
```suggestion
`repeat()` is a CSS function available to the CSS Grid template properties that allows us to define a number of rows or columns and the size we want them to be without having to manually type out each individual track's size. For example, in our setup above:
```